.gitignore

```diff
@@ -4,3 +4,5 @@ moduleIndex.json
 last-moduleIndex.json
 generated-moduleIndex.json
 .DS_Store
+avm/res/cdn/profile/Test-CDN.ps1
```

Comment:
Are these leftovers?

Reply:
We can ignore these changes; initially I had my custom local test file in the CDN folder so I added it to the gitignore to avoid uploading it but then I moved it to another location outside the repo like you suggested